tests/RobotFramework/tests/mqtt/custom_sub_topics_tedge-mapper-az.robot

```diff
@@ -11,20 +11,20 @@ Test Tags           theme:mqtt    theme:az
 
 
 *** Test Cases ***
-Publish measurements to subscribed topic
-    Execute Command    tedge mqtt pub tedge/measurements/child '{"temperature": 10}'
-    Should Have MQTT Messages    az/messages/events/#
+Publish events to subscribed topic
+    Execute Command    tedge mqtt pub te/device/main///e/event-type '{"text": "Event"}'
+    Should Have MQTT Messages    az/messages/events/device:main/e/#
 
 Publish measurements to unsubscribed topic
-    Execute Command    tedge mqtt pub tedge/measurements '{"temperature": 10}'
+    Execute Command    tedge mqtt pub te/device/main///m/ '{"temperature": 10}'
     Sleep    5s    reason=If a message is not published in 5s, it will never be published.
-    Should Have MQTT Messages    az/messages/events/#    minimum=0    maximum=0
+    Should Have MQTT Messages    az/messages/events/device:main/m/   minimum=0    maximum=0
```

Comment:
Adding `#` is slightly better as it ensures no messages published on any measurement topics stronger regardless of type.

```suggestion
    Should Have MQTT Messages    az/messages/events/device:main/m/#   minimum=0    maximum=0
```